lib/clients/apifutebol/client.rb

```diff
@@ -25,15 +25,13 @@ def match(id) # rubocop:disable Metrics/AbcSize
       request = Net::HTTP::Get.new uri
 
       response = http.request request # Net::HTTPResponse object
-      if response.is_a?(Net::HTTPSuccess)
-        # Parse the JSON response
-        parsed_response = JSON.parse(response.body)
-        return ImportMatch.new(parsed_response)
-
-      else
-        puts "Request failed with HTTP status code: #{response.code}"
-        puts "Response Body: #{response.body}"
+      unless response.is_a?(Net::HTTPSuccess)
+        raise "Request failed with HTTP status code: #{response.code}\nResponse Body: #{response.body}"
       end
+
+      # Parse the JSON response
+      parsed_response = JSON.parse(response.body)
+      return ImportMatch.new(parsed_response)
```

Comment:
No need for explicit return here 😉 

```rb
      ImportMatch.new(parsed_response)
```

Reply:
Hahahahaha... habits do die hard :smile: